content/install-keras-environment-from-scratch.md

```diff
@@ -0,0 +1,174 @@
+Title: Install a Keras environment from scratch
```

Comment:
I would call it "Set up a Keras Deep Learning environment from scratch". At least "set up": you don't install an environment.